dask_kubernetes/core.py

```diff
@@ -315,7 +318,11 @@ class KubeCluster(SpecCluster):
     env: Dict[str, str]
         Dictionary of environment variables to pass to worker pod
     host: str
-        Listen address for local scheduler.  Defaults to 0.0.0.0
+        Listen address for local scheduler. Only used when deploy_mode == "local". Defaults to
+        0.0.0.0
+    nodeport_host: str
+        When scheduler_service_type == ``"NodePort"``, override KubeCluster's default behaviour
+        of listing nodes by specifying a nodeport_host.
```

Comment:
May I suggest a more user focused docstring here. 

```suggestion
    nodeport_host: str
        When scheduler_service_type == ``"NodePort"``, select a specific node IP to connect to.
        When this is not set the node IP will be autodetected.
```